packages/cli/package.json

```diff
@@ -22,7 +22,7 @@
   ],
   "type": "module",
   "bin": {
-    "configu": "./dist/run.cjs"
+    "configu2": "./dist/run.cjs"
```

Comment:
?

Reply:
i tested it locally and wanted to be sure I'm using this cli and not my already-installed one.  reverting it now.